dev/reference/tools/mydumper.md

```diff
@@ -18,15 +18,11 @@ It can be [downloaded](/dev/reference/tools/download.md) as part of the Enterpri
 
 + This tool uses the hidden `_tidb_rowid` column of TiDB to optimize the performance of concurrently exporting data from a single table.
 
-## How to use Mydumper?
+## Usage
 
 ### New parameter description
 
-```
-  -z, --tidb-snapshot: Set the tidb_snapshot to be used for the backup.
-                       Default: The current TSO (Position field from SHOW MASTER STATUS).
-                       Accepts either a TSO or valid datetime. For example: -z "2016-10-08 16:45:26"
-```
+`-z` or `--tidb-snapshot`: Sets the `tidb_snapshot` to be used for the backup. The default value is the current TSO (the `Position` field from `SHOW MASTER STATUS`). This parameter can be accepted either as a TSO or a valid `datetime`. For example: `-z "2016-10-08 16:45:26"`.
```

Comment:
- Keep the capitalization style consistent in this doc.
- "This parameter can be accepted" is a little bit weird to me.
```suggestion
`-z` or `--tidb-snapshot`: sets the `tidb_snapshot` to be used for the backup. The default value is the current TSO (the `Position` field output from `SHOW MASTER STATUS`). Set this parameter to the TSO or a valid `datetime` such as `-z "2016-10-08 16:45:26"`.
```